### ISSUE TITLE
Add a screenshot to the Issue to help identify the problem

### ISSUE THREAD

**Issue by Th3S4mur41** [github/accessibility-scanner#106]
This is the example of the description of an issue created:

> An accessibility scan flagged the element <p> on https://*** because elements must meet minimum color contrast ratio thresholds. Learn more about why this was flagged by visiting https://dequeuniversity.com/rules/axe/4.11/color-contrast?application=playwright.
> 
> To fix this, ensure the contrast between foreground and background colors meets wcag 2 aa minimum contrast ratio thresholds.
> 
> Specifically:
> 
> Fix any of the following:
> 
> Element has insufficient color contrast of 4.42 (foreground color: #878787, background color: #222222, font size: 15.0pt (20px), font weight: normal). Expected contrast ratio of 4.5:1

The description doesn't make it easy to identify the problematic element on large pages.
In the example above, there are a lot of `<p>` elements on the page.

A screenshot, would help narrow down the culprit

**Comment by helen** [comment]
We've been trying this out internally, including how to make sure reasonable alt text gets included with it back 😅 hope to have something to share in the fairly near future!

**Comment by Th3S4mur41** [comment]
Just tested it... Works like a charm and absolutely helpful 👍
Thanks for the quick improvement.
